certs/service.go

```diff
@@ -78,12 +79,14 @@ func (cs *certsService) IssueCert(ctx context.Context, domainID, token, clientID
 		return Cert{}, errors.Wrap(ErrFailedCertCreation, err)
 	}
 
+	_, err = cs.certsRepo.Save(ctx, cert)
```

Comment:
Check this error here and then return:
```suggestion
return Cert{
		SerialNumber: cert.SerialNumber,
		Certificate:  cert.Certificate,
		Key:          cert.Key,
		ExpiryTime:   cert.ExpiryTime,
		IssuingCA:    cert.IssuingCA,
		CAChain:      cert.CAChain,
		ClientID:     cert.ClientID,
	}, nil
```